data-sources/sourcecred/output/sourcecred-0xEdd000B7Db3cb8931d4E0cb1D0DBe6B947Ceb09A.csv

```diff
@@ -0,0 +1,26 @@
+week_end,sourcecred_score
+2022-05-29,6.388218329908233
+2022-06-05,17.32659447602243
+2022-06-12,32.900162447772864
+2022-06-19,25.601593328752386
+2022-06-26,35.27082633521167
+2022-07-03,6.6415714473380305
+2022-07-10,18.071806283104202
+2022-07-17,17.994233153289187
+2022-07-24,17.40126086317772
+2022-07-31,17.550842286542277
+2022-08-07,19.33882579165539
+2022-08-14,24.958849504244426
+2022-08-21,19.22577799506177
+2022-08-28,20.95379064413659
+2022-09-04,19.042056140019888
+2022-09-11,13.877407707761318
+2022-09-18,13.408322174164748
+2022-09-25,35.93574023131355
+2022-10-02,23.887490905949548
+2022-10-09,32.839732475598694
+2022-10-16,27.18149361003286
+2022-10-23,37.622205160678234
+2022-10-30,16.679218880272227
+2022-11-06,26.7748052575784
+2022-11-13,6.631119484103152
```

Comment:
@johnmark13 Maybe not generate the row of data for a week until the week's end date (Sunday midnight) has been passed?

Reply:
👍